pandas/tseries/tests/test_resample.py

```diff
@@ -419,25 +419,32 @@ def test_agg_misc(self):
             assert_frame_equal(result, expected, check_like=True)
 
         # series like aggs
-        expected = pd.concat([t['A'].sum(),
-                              t['A'].std()],
-                             axis=1)
-        expected.columns = ['sum', 'std']
-
         for t in [r, g]:
-            result = r['A'].agg({'A': ['sum', 'std']})
+            result = t['A'].agg({'A': ['sum', 'std']})
+            expected = pd.concat([t['A'].sum(),
+                                  t['A'].std()],
+                                 axis=1)
+            expected.columns = ['sum', 'std']
+
+            assert_frame_equal(result, expected, check_like=True)
+
+            expected = pd.concat([t['A'].agg(['sum', 'std']),
+                                  t['A'].agg(['mean', 'std'])],
+                                 axis=1)
+            expected.columns = pd.MultiIndex.from_tuples([('A', 'sum'),
+                                                          ('A', 'std'),
+                                                          ('B', 'mean'),
+                                                          ('B', 'std')])
+            result = t['A'].agg({'A': ['sum', 'std'], 'B': ['mean', 'std']})
```

Comment:
```
In [4]: g['D'].agg({'D': np.sum, 'result2': np.mean})
Out[4]: 
          result2   D
A   B                
bar one  1.000000   1
    two  4.000000   8
foo one  3.000000   6
    two  4.333333  13

In [5]: g['D'].agg({'D': np.sum, 'C': np.mean})
Out[5]: 
                C   D
A   B                
bar one  1.000000   1
    two  4.000000   8
foo one  3.000000   6
    two  4.333333  13
```

(groupby and resample work the same).

That was my point though. maybe we *should* raise here as the user could think they are actually operating on something they are not.